wmigroup/wmigroup.resource.ps1

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+[CmdletBinding()]
+param(
+    [ValidateSet('List','Get','Set','Test')]
+    $Operation = 'List',
+    [Parameter(ValueFromPipeline)]
+    $stdinput
+)
+
+$ProgressPreference = 'Ignore'
+$WarningPreference = 'Ignore'
+$VerbosePreference = 'Ignore'
+
+if ($Operation -eq 'List')
+{
+    $clases = Get-CimClass
+    
+    foreach ($r in $clases)
+    {
+        $version_string = "";
+        $author_string = "";
+        $moduleName = "";
+
+        $propertyList = @()
+        foreach ($p in $r.CimClassProperties)
+        {
+            if ($p.Name)
+            {
+                $propertyList += $p.Name
+            }
+        }
+
+        $namespace = $r.CimSystemProperties.Namespace.ToLower().Replace('/','\')
```

Comment:
If we _do_ want to make the type name compliant with the current pattern for DSC Resource type names, I think we'd have to use `.` instead of `\`.

Reply:
I agree. Updated PR to use `.` instead of `\`.